components/script/dom/htmlmetaelement.rs

```diff
@@ -85,6 +90,28 @@ impl HTMLMetaElement {
             }
         }
     }
+
+    /// https://html.spec.whatwg.org/multipage/#meta-referrer
+    fn apply_referrer(&self) {
+        /*todo - I think this chould only run if document's policy hasnt yet
+        been set. unclear - see:
+        https://html.spec.whatwg.org/multipage/#meta-referrer
+        https://w3c.github.io/webappsec-referrer-policy/#set-referrer-policy
+        */
+        let doc = document_from_node(self);
+        if let Some(head) = doc.GetHead() {
+            if head.upcast::<Node>().is_parent_of(self.upcast::<Node>()) {
+                let element = self.upcast::<Element>();
+                if let Some(content) = element.get_attribute(&ns!(), &atom!("content")).r() {
+                    let content = content.value();
+                    let content_val = content.trim();
```

Comment:
I don't like this double let, but "let content = content.value().trim();' throws "borrowed value does not live long enough."

Better way?